client/src/components/admin/tables/ProjectRow.tsx

```diff
@@ -91,13 +111,39 @@ const ProjectRow = ({ project, idx, checked, handleCheckedChange }: ProjectRowPr
                     ></input>
                 </td>
                 <td>{project.name}</td>
+                <td className="flex justify-center">
+                    <button
+                        onClick={() => {
+                            setShowFlags(true);
+                        }}
+                    >
+                        {matchingFlag ? (
```

Comment:
Instead of doing this:

```jsx
matchingFlag ? (...code) : ''
```

You can just do:

```jsx
matchingFlag && (...code)
```

If `matchingFlag` is false, it'll short circuit and won't render anything. I do it in [`AdminToolbar.tsx`](https://github.com/hackutd/jury/blob/53c28b6c4d7291ef3f3e8fa3f1210506ee240d3d/client/src/components/admin/AdminToolbar.tsx#L22).

Reply:
im not marking all instances of this but you should change all of them